parlai/utils/bpe.py

```diff
@@ -124,8 +127,20 @@ def add_cmdline_args(argparser):
             hidden=True,
             help='add prefix space before encoding',
         )
+        parser.add_argument(
+            '--bpe-dropout',
+            type=float,
+            default=None,
+            help='Use BPE dropout during training.',
+        )
         return parser
 
+    def set_training_mode(self, mode: bool):
```

Comment:
That was intentional. BPE dropout is just one training time decision you might make.

Reply:
If this can take multiple values beyond just a setting BPE to true, either it should be changed away from a bool to an enum or `mode` should be `use_bpe` or something. 

(All kind of nits, but from a "someone else looking at the code quickly figuring out what's supposed to happen here" perspective...)